### DISCUSSION TITLE
Gamepad for Nintendo Switch

### DISCUSSION THREAD

**Discussion by gdsports** [NicoHood/HID#282] [Ideas]
https://github.com/gdsports/HID/tree/horipad/examples/NSGamepad

Inspired by various NS gamepad projects using LUFA and AVR boards. SAMD has more capability such as capactive touch. Not much testing so far but buttons work.

**Comment by gdsports** [comment]
This has a new home at https://github.com/gdsports/NSGadget_HID

**Comment by sparten9999** [comment]
i gave the above link a try and just get "not declared in this scope" errors in all examples
nsgadget gives 'NSGamepad' was not declared in this scope
nsgamepad gives 'NSGAMEPAD_DPAD_CENTERED' was not declared in this scope
nsgamepad demo gives nsgadget gives 'NSGamepad' was not declared in this scope
nskeyboard gives 'NSGAMEPAD_DPAD_CENTERED' was not declared in this scope
